deduplication/src/main/java/gov/cdc/nbs/deduplication/seed/mapper/MpiPersonMapper.java

```diff
@@ -150,16 +150,27 @@ List<Telecom> mapPhones(String phoneString) {
         }).orElseGet(() -> new ArrayList<>());
   }
 
-  /** Returns the first Drivers License entry found, or null */
+  /** Returns the first Drivers-License entry found, or sets to empty string */
   DriversLicense mapDriversLicense(String driversLicenseString) {
-    return tryParse(
-        driversLicenseString,
-        new TypeReference<List<DriversLicense>>() {
-        }).orElseGet(() -> new ArrayList<>())
-        .stream()
-        .findFirst()
-        .orElse(null);
+    List<DriversLicense> licenses = tryParse(
+            driversLicenseString,
+            new TypeReference<List<DriversLicense>>() {
+            }).orElseGet(() -> new ArrayList<>());
+
+    // Get the first drivers-license or set it to null
+    DriversLicense license = licenses.stream().findFirst().orElse(null);
+
+    // If the license is null, return a default one
+    if (license == null) {
```

Comment:
I think it's still ok to return null in the case of no drivers license entry at all.